common/ast/src/ast/expr.rs

```diff
@@ -689,6 +693,17 @@ impl<'a> Display for Expr<'a> {
                 write_comma_separated_list(f, exprs)?;
                 write!(f, "]")?;
             }
+            Expr::Interval { expr, unit, .. } => {
```

Comment:
I prefer to move it to `Literal` since there is already `CurrentTimestamp`.

Reply:
We can discuss with @sundy-li to find better way to dowith it.

Reply:
It might not be literal if we want to have column-based Interval.

So we choose to have `interval expr unit`, surely if it's literal it may make life easier.

refer: [Doris](https://doris.apache.org/sql-manual/sql-functions/date-time-functions/date_add.html#date-add)

Reply:
Will it take a bad effect on the query `select Year, count(1) from ontime group by Year `  ?

Reply:
If we put token like YEAR, MONTH.. as reserved indent, it will take a bad effect. In fact, I do this at beigin which can't pass the stateful-test. So, I recoginize them as reserved function.

Reply:
If we set `YEAR` to reserved ident, it will take a bad effect. 

Now I submit a new pr #5490 to delete `YEAR` from reserved function. It is not unnecessary.